apps/pollbook/backend/src/backup_worker.ts

```diff
@@ -128,7 +130,7 @@ export async function getBackupPaperChecklistPdfs(
     });
 
   const voterCountByParty = store
-    .getAllVotersSorted()
+    .getAllVotersSorted({ matchConfiguredPrecinctId: true })
```

Comment:
I think this will just always return an empty list of voters before the machine is configured with a precinct, it looks like we wait for the election to be configured but not the precinct before we start trying to export the backup, but maybe we should also not be exporting unless configuredPrecinctId is defined? cc @adghayes